### PR TITLE
Fix issue 31

### PR DESCRIPTION
**Pull request by viclovsky**
https://github.com/allure-framework/allure-kotlin/issues/31

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -32,14 +33,14 @@ allprojects {
...
     apply(from = "$gradleScriptDir/bintray.gradle")
+    apply(plugin = "com.github.dcendents.android-maven")
 }
 
 configure(subprojects
         .filter { !it.name.contains("android") }
         .filter { it.parent?.name != "samples" }
 ) {
     apply(plugin = "org.jetbrains.kotlin.jvm")
-    apply(from = "$gradleScriptDir/maven-publish.gradle")
```
Comment (kamildziadek) [inline build.gradle.kts:42]: Why was this removed? It nicely configures the pom of pure kotlin libraries. I would say that it should be kept and 
`apply(plugin = "com.github.dcendents.android-maven")` should only be applied to android module. It seems weird that we are using the `android-maven` plugin for kotlin like modules

Comment (viclovsky) [reply build.gradle.kts]: @kamildziadek, thanks a lot, good catch.
